src/test/regress/multi_1_schedule

```diff
@@ -33,6 +33,7 @@ test: ref_citus_local_fkeys
 test: alter_database_owner
 test: distributed_triggers
 test: create_single_shard_table
+test: single_shard_table_udfs
```

Comment:
Can we try to run this test in parallel to some other in this schedule?

Reply:
I believe it's perfectly ok to parallelize the tests with unrelated files. Right now it takes 0.1 second to run but in the future we might have more lines in that test file?